packages/grid/_modules_/grid/components/panel/GridColumnsPanel.tsx

```diff
@@ -131,6 +133,7 @@ export function GridColumnsPanel() {
               <FormControlLabel
                 control={
                   <Switch
+                    disabled={column.hidable === false}
```

Comment:
Should we show it in a disabled state or not show the option? Or even better, add a prop to `GridColumnsPanel` to allow to configure this?

![image](https://user-images.githubusercontent.com/42154031/146044975-3281cc9b-90d7-4ad1-837c-27b85b504c60.png)

Reply:
I think we should show it in a disabled state because if we remove the column, it will be impossible to reorder columns by drag and drop.